README.md

```diff
@@ -70,8 +70,9 @@ If you have downloaded Zettlr, [head over to our website](https://docs.zettlr.co
 ## Contributing
 
 Zettlr is an [Electron](https://www.electronjs.org/)-based app, so to start developing, you'll need to have:
+
 1. A [NodeJS](https://nodejs.org/)-stack on your computer installed. Make sure it is Node 12+ (`lts/erbium`). To test what version you have, try `node -v`.
```

Comment:
If you feel up to it, would you mind fixing this line too? 

'A NodeJS-stack installed' or 'installed on your computer' is more grammatically correct. If not I'll do it at some point.

Reply:
It's probably better to keep this scope smaller to reduce approval time.

Reply:
This change doesn't affect approval time, as it's trivial ;)

Reply:
Even more trivial with this smaller change. :laughing: